templates/aspnet-core/files/Data/MyDbContext.cs

```diff
@@ -0,0 +1,15 @@
+using Microsoft.EntityFrameworkCore;
+
+namespace PlatformshAspNetCore.Data
+{
+    public class MyDbContext : DbContext
```

Comment:
I think in this case this is extending `DbContext` from `Microsoft.EntityFrameworkCore` into the `PlatformshAspNetCore` namespace so it can be used by the application. This could be done in `Startup.cs`, but I think this is the recommended way to separate things in case you wanted to modify `MyDbContext`.